apollo-backend/src/main/java/io/logz/apollo/deployment/DeploymentHandler.java

```diff
@@ -36,6 +37,7 @@
 
     private static final Logger logger = LoggerFactory.getLogger(DeploymentController.class);
 
+    private ApolloNotifications apolloNotifications;
```

Comment:
should be final

Reply:
you should revert changes in this class as you moved `apolloNotifications.notify(...)` to KubernetesHandler and apolloNotifications is not used anymore